apis/Google.Cloud.Firestore/Google.Cloud.Firestore/TransactionOptions.cs

```diff
@@ -24,27 +25,53 @@ public sealed class TransactionOptions
         /// <summary>
         /// The transaction options that are used if nothing is specified by the caller.
         /// </summary>
-        public static TransactionOptions Default { get; } = new TransactionOptions(5);
+        public static TransactionOptions Default { get; } = new TransactionOptions(5, null);
 
         /// <summary>
         /// The number of times the transaction will be attempted before failing.
         /// </summary>
         public int MaxAttempts { get; }
```

Comment:
Could we deprecate this one instead of ignoring the one in RetrySettings?

Reply:
Feels a bit like overkill if most people won't need to touch the timing - creating a retry setting is relatively complex. Happy to chat about it though.